src/backend/ai-reviewer-mock-api/src/test/java/ca/bc/gov/open/jag/aireviewermockapi/controller/ParentControllerTest.java

```diff
@@ -2,43 +2,32 @@
 
 import ca.bc.gov.open.jag.aireviewermockapi.model.DocumentReady;
 import org.junit.jupiter.api.*;
-import org.mockito.ArgumentMatchers;
-import org.mockito.Mock;
-import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.client.RestTemplate;
 
 import java.math.BigDecimal;
 
-import static org.mockito.ArgumentMatchers.any;
-
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
You didn't change this line, but may I ask why per_class?  This is unusual.  My understanding is with this setting is that tests are no longer atomic - the output of one can affect the results of a different test.  In this class it doesn't matter since there is only one test anyway, but then, why?

Reply:
It's been a while. But this one allows us to use the @BeforeAll annotation. However I think we have just been adding it to every class out of habit. In this case it isn't necessary.